DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -43,6 +43,7 @@ public function load(array $configs, ContainerBuilder $container)
         $config = $this->processConfiguration($configuration, $configs);
 
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('matcher.xml');
```

Comment:
the services being private, they will be removed when the container is built, so this should have 0 runtime impact